web/turkey/survey/static/survey/js/auditors/visibility_changes.js

```diff
@@ -0,0 +1,17 @@
+// helper script file for focus_change auditors
```

Comment:
This was a curious case, because these auditors are watching a global variable that denotes a change in focus. We want an auditor to log something when an event occurs, and this is the event detection. I just wasn't sure how to incorporate this variable into an auditor. 

I thought about sticking the helper js code into the `$(document).ready(...)` part of the auditors, but we would have the same script load/execute multiple times if we use more than one of those auditors I think. Not sure.

I would prefer to have the helper script somehow integrated to be within the auditors themselves as well, but I think I will need your help with this.